### PR TITLE
add Dymension' channels info: Celestia, CosmosHub, Kuj, Stride, Axl

### PR DESCRIPTION
**Pull request by svv28**

### CODE REVIEW

File: _IBC/dymension-axelar.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "dymension",
```
Comment (JeremyParish69) [inline _IBC/dymension-axelar.json:4]: not following alphabetical order (axelar must be chain_1)

Comment (JeremyParish69) [reply _IBC/dymension-axelar.json:4]: ditto for celestia and cosmoshub

Comment (svv28) [reply _IBC/dymension-axelar.json]: sorry, missed that rule in the description!
fixed.
take a look, plz

Comment (JeremyParish69) [reply _IBC/dymension-axelar.json]: Oh, these two file names also need to be swapped

Comment (svv28) [reply _IBC/dymension-axelar.json]: deleted duplicated file - hub-dym,
renamed:
 axl-dym
 cel-dym

**Pull request merged**